src/util.c

```diff
@@ -967,3 +968,46 @@ int utilTest(int argc, char **argv, int accurate) {
     return 0;
 }
 #endif
+
+int createPipe(int fds[2], int read_flags, int write_flags) {
+#if defined(__linux__) || defined(__FreeBSD__) 
+    int flags = O_CLOEXEC;
```

Comment:
We always set this for linux because it does no harm and it's no extra cost? Maybe explain this in a comment.

Reply:
Yes, it is like a convention without extra side effects, I'll add a comment fot it.